.github/actions/changelogs-generator/action.yml

```diff
@@ -0,0 +1,59 @@
+name: Generate Changelogs
+description: 'Bump Version and Generate Changelogs'
+inputs:
+  repo_name:
+    description: 'Name of the repo in which action is called'
+    required: true
+  version:
+    description: 'New version number'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        repository: ${{ inputs.repo_name }}
+    - name: setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+
+    - name: Install package
+      shell: bash
+      run: pip install python-semantic-release
+
+    - name: Git setup
+      shell: bash
+      run: |
+        git config --global user.email "actions@github.com"
+        git config --global user.name "github-actions"
+
+    # Make sure you've added version_variable in setup.cfg 
+    - name: Update Version
+      shell: bash
+      run: semantic-release version
+
+    - name: Install pandoc
+      shell: bash
+      run: brew install pandoc
```

Comment:
I always thought brew was specific to Mac.  I see they support Linux also, but wouldn't it be better to use a more native package manager?

Reply:
Will try to find out an alternative